source/key.js

```diff
@@ -157,6 +166,22 @@ const importSymKeyFromBase64 = (base64Key, algorithm = AES_GCM) =>
         [ENCRYPT, DECRYPT],
     );
 
+/**
+ * Import from hexadecimal encoded raw key to CryptoKey.
+ * An imported key should never be extractable.
+ *
+ * @param {String} hexadecimal - hexadecimal encoded key
+ * @param algorithm - cryptographic algorithm
+ * @returns {Promise} Resolves to the key as a CryptoKey.
+ */
+const importSymKeyFromHexadecimal = (hexadecimal, algorithm = AES_GCM) =>
+    crypto.subtle.importKey(
+        'raw',
+        convertHexadecimalToArrayBuffer(hexadecimal),
+        algorithm,
+        false,
+        [ENCRYPT, DECRYPT],
+    );
 
 const importKey = (key) => {
```

Comment:
I'm not sure how we use this and why such a function was needed, but I would always prefer the explicit import (i.e. `importSymKeyFromHexadecimal`) over this function; there is quite a lot of magic hidden in here, for every key type something different happens and it's opaque for the caller.

Reply:
I assume the `importKey` existence is based on historical and personal preferences (code organization) in terms of structuring the API for this library 🤷‍♂️

> there is quite a lot of magic hidden in here

I'd argue on that since it's one `switch` block (object literal might be better suited) with a mapping to the `KEYTYPES` config.

Considering that `importSymKeyFromHexadecimal` is exported and the fact that refactoring the `keys.js` code is out of the scope of this PR, I'll avoid any changes that might affect other projects using the `importKey` method.